src/Memory/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(Memory)
+add_library(Memory STATIC)
+
+target_compile_options(Memory PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_include_directories(Memory PUBLIC
+        ${CMAKE_CURRENT_LIST_DIR}/include)
+
+target_include_directories(Memory PRIVATE
+        ${CMAKE_CURRENT_LIST_DIR})
+
+target_sources(Memory PUBLIC
+        include/Memory/MemoryInfoListener.h
+        include/Memory/MemoryInfoProducer.h)
+
+target_sources(Memory PRIVATE
+        MemoryInfoProducer.cpp
+        MemoryUtils.cpp
+        MemoryUtils.h)
+
+target_link_libraries(Memory PUBLIC
+        GrpcProtos
+        OrbitBase
+        CONAN_PKG::abseil)
+
+add_executable(MemoryTests)
+
+target_compile_options(MemoryTests PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_sources(MemoryTests PRIVATE MemoryUtilsTest.cpp)
```

Comment:
The naming conventions is to get a header file and have a test named after it:
MemoryInfoListener.h -> MemoryInfoListenerTests.cpp